api/libproject/devstart.rb

```diff
@@ -2020,20 +2022,36 @@ def connect_to_cloud_db(cmd_name, *args)
   common = Common.new
   op = WbOptionsParser.new(cmd_name, args)
   op.add_option(
```

Comment:
If I were looking at the help text quickly, I'd probably be looking for an option like `--read-only` or `--access=read-only` rather than `--db-user` (which is a bit of an implementation detail).

Reply:
I added more to the flag doc. The problem with that approach is that we have existing functionality which supports signing in as --root instead. So the --readonly proposal means we now have mutually exclusive flags with confusing interactions, since the implementations are all driven by the database user e.g. --readonly, --db-user or , --readonly and --root

I'd prefer to leave this general and add documentation. Also - I feel readonly should be the default, so I don't like it as a flag. --no-readonly maybe (suffers from same issues).